docs/backlog-management.md

```diff
@@ -0,0 +1,79 @@
+# Issue Management for OpenTelemetry
+
+It's an important community goal for OpenTelemetry that our members find the backlogs to be responsive, and easy to take part in. Having a shared methodology for processing issues helps new members understand the current state of each project.
+
+SIGs are encouraged to experiment with labels and backlog management procedures. This document only covers the bare bones of issue management which should work the same across all SIGs, to help maintain a responsive backlog and allow us to track work across all projects in a similar manner.
+
+## New Issues
+
+Flow:
+* An issue is filed by a community member.
+* A maintainer responds and asks clarifying questions.
+* The maintainer processes the issue as a Code Change, a Discussion, or resolves it quickly.
+
+Every new issue should be responded to quickly by a Maintainer, with a greeting and any needed clarifying questions. If assessing the issue requires more information, the Triager collaborates with the OP until workability is determined. If the ticket is a help request with an existing answer, please link to the answer before closing the ticket.
+
+## Code Changes
+
+Flow:
+* A new issue is filed, proposing a code change.
+* Maintainer labels the issue as `up-for-grabs`; reviewers are assigned.
+* `up-for-grabs` becomes `wip` once a Collaborator is assigned.
```

Comment:
How about adding an assignee and then leveraging a Github project board? It can move the issue automatically to an in-progress section (concerned about process overhead here).

Reply:
Or people just mention themselves in comments. Not everybody is OpenTelemetry project member